### DISCUSSION TITLE
Need some help

### DISCUSSION THREAD

**Discussion by tandem97** [bluenviron/gortsplib#141] [Questions]
Hello everyone!
 I want to read rtsp from server and start serving this rtsp. I explore examples, the most reliable is https://github.com/aler9/gortsplib/blob/main/examples/client-read-republish/main.go but i want to serve not send rtsp after reading it.
 Thank you for help!

**Comment by aler9** [comment]
Hello, i've added an example that shows how to pull a stream from an existing server and serve it with a server:

https://github.com/aler9/gortsplib/blob/main/examples/proxy/main.go
